addon/core/handler.ts

```diff
@@ -312,7 +317,24 @@ export default class TableHandler {
 
   private _reinitColumnsAndRows(columns: Column[]): void {
     this.rows = [];
-    this.columns = columns;
+
+    let shouldRedraw = false;
+
+    columns.forEach((column) => {
+      let existingColumn = this.columns.find((c) => c.definition.key === column.definition.key);
+
+      if (existingColumn) {
+        existingColumn = column;
+      } else {
```

Comment:
I'm not sure what `existingColumn` is used for ? you assign it to `column` if it is non-null but you don't use it anywhere else in the function's scope ? 🤔

Reply:
All good !